torch/_inductor/codegen/cpp.py

```diff
@@ -1399,10 +1409,7 @@ def store(self, name, index, value, mode=None):
         elif (V.graph.get_dtype(name) in [torch.uint8]) and (
             opt_ctx.is_store_float_as_uint8
```

Comment:
Do we still need this `opt_ctx.is_store_float_as_uint8` flag here?

Reply:
Sure and removed. I think we can remove `opt_ctx.is_store_float_as_uint8` since this only uint8 data type case we support.